### ISSUE TITLE
metric extrinsic?

### ISSUE THREAD

**Issue by dreamer1031** [ByteDance-Seed/Depth-Anything-3#73]
i use the model -- DA3NESTED-GIANT-LARGE to infer some pictures like this:
def infer(model, base_imgpath_lists):
    with torch.no_grad():
        res = model.inference(
            base_imgpath_lists, export_dir=None, process_res_method="lower_bound_resize", infer_gs=False
        )

    predictions = res
    poses =  predictions.extrinsics #world to camera
    depths = predictions.depth
    is_metric = predictions.is_metric
    scale_factor = predictions.scale_factor
    print(is_metric)
    print(scale_factor)
    N, H, W = depths.shape
    camera_poses = np.zeros((N, 4, 4))
    for i in range(N):
        c2w = np.linalg.inv(_as_homogeneous44(poses[i]))  # (4,4)
        camera_poses[i] = c2w
        print(c2w)

    return camera_poses

the is_metric is 1 and the scale_factor is 3.6158194541931152
do i need to use the c2w * scale_factor to get the metric pose?? or the pose is already the metric pose?

**Comment by haotongl** [comment]
@dreamer1031 Hi, thanks for your attention. It's metric pose.
